### PR TITLE
0.1/622 convert html to a blade skeleton

### PR DESCRIPTION
**Pull request by charlesstrange2**
Initial template conversion. Magical dynamic fronted stuff to come.

### CODE REVIEW

File: resources/views/service/dashboard.blade.php [hunk 1]
```diff
@@ -5,9 +5,21 @@
...
+            </a>
+        </li>
+        <li>
+            <a href="">
+                <i class="fa fa-user" aria-hidden="true"></i> Check, update or print information for one family
+            </a>
+        </li>
+        <li>
+            <a href="">
+                <i class="fa fa-users" aria-hidden="true"></i> Print collection sheets for all families<
```
Comment (katjam) [inline resources/views/service/dashboard.blade.php]: stray `<` ?

File: resources/views/service/registration.blade.php [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                <div>
+                    <h2>Children signed up:</h2>
+                    <table>
+                        <tr>
+                            <td>Age</td>
+                            <td>Month/Year</td>
+                            <td>Info</td>
+                        </tr>
+                        <tr>
+                            <td>7</td>
```
Comment (katjam) [inline resources/views/service/registration.blade.php:55]: Stubs?

File: resources/views/service/registration.blade.php [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+            </div>
+            <div class="col">
+                <div>
+                    <label for="cc-id">CC ID number:</label>
+                    <input type="text" id="cc-id">
+                </div>
+                <div>
+                    <h2>Reason for receiving Rose Vouchers <span><i class="fa fa-info-circle"
+                                                                    aria-hidden="true"></i></span></h2>
+                    <div class="user-control">
```
Comment (katjam) [inline resources/views/service/registration.blade.php:83]: Would these be better as a list?

File: routes/api.php [hunk 4]
```diff
@@ -13,6 +13,8 @@
 |
 */
 
+/*
 Route::middleware('auth:api')->get('/user', function (Request $request) {
```
Comment (katjam) [inline routes/api.php]: Intentionally commented out?

**Review by katjam — APPROVED** [review]
Few small comments - but once considered. Good to go. :+1:

File: resources/views/service/registration.blade.php (hunk 2 above)
Comment (charlesstrange2) [reply resources/views/service/registration.blade.php:55]: yup, that's going to be flattened by a vue component.

File: resources/views/service/registration.blade.php (hunk 3 above)
Comment (charlesstrange2) [reply resources/views/service/registration.blade.php:83]: There will only be 2, HF or "other"; Arcane ICO/DPA reasons for not wanting custom fancylists.

File: routes/api.php (hunk 4 above)
Comment (charlesstrange2) [reply routes/api.php]: yep, will comment on it.

**Pull request merged**